src/grammar.json

```diff
@@ -877,14 +912,15 @@
         "#realNormalCard#",
         "#realNormalCard#",
         "#realNotableCard#",
+        "#realObscureCard#",        
         "#realPowerfulCard#",
         "#realBadCard#"
     ],
 
     "fakeCard":
     [
-        "Hurloon Wrangler",
-        "Gleemax",
+        "Every Last Vestige Shall Rot",
```

Comment:
adding any real cards that won't load in mtgcardfinder to the fakeCard section for now